src/plugins/cal-events/cal-events.js

```diff
@@ -436,6 +457,10 @@ $document.on( "keydown", ".wb-calevt-cal .cal-days td > ul li", function( event
 	}
 } );
 
+$document.on( "focusout", ".wb-calevt-cal .cal-days td > ul > li > a", function( event ) {
```

Comment:
Can we use a different method to detect a focus out, like with our overlay plugin.

However, the CSS selector for this type of event is too much precise.